### ISSUE TITLE
Fastly 0.0.3 code isn't on PyPI

### ISSUE THREAD

**Issue by LotharSee** [fastly/fastly-py#19]
Fastly 0.0.3 isn't available on the index.

The web page exists: https://pypi.python.org/pypi/fastly/0.0.3
But the code isn't there.

```
$ pip install fastly==0.0.3

Collecting fastly==0.0.3
  Could not find a version that satisfies the requirement fastly==0.0.3 (from versions: 0.0.2)
  No distributions matching the version for fastly==0.0.3
```

**Comment by thorduri** [comment]
Poked on Fastly's IRC yesterday (2015-11-24) and was told that someone was prodded internally.

A bit sad that this has been outstanding since March (ditto other issues/PRs).

**Comment by thommahoney** [comment]
This is now fixed in version 0.0.5
```
› pip install fastly==0.0.5
Collecting fastly==0.0.5
  Downloading fastly-0.0.5.tar.gz
Building wheels for collected packages: fastly
  Running setup.py bdist_wheel for fastly ... done
  Stored in directory: /Users/thom/Library/Caches/pip/wheels/d2/45/29/c34f492fe25a7fc16c60cc88c24530e2fdfd4058b633468630
Successfully built fastly
Installing collected packages: fastly
  Found existing installation: fastly 0.0.2
    Uninstalling fastly-0.0.2:
      Successfully uninstalled fastly-0.0.2
Successfully installed fastly-0.0.5
```

**Comment by thorduri** [comment]
Thanks @thommahoney !
